tests/CMakeLists.txt

```diff
@@ -287,3 +287,31 @@ ENDIF()
 # Set up the default attest configuration file:
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/attest.conf.in
   ${CMAKE_BINARY_DIR}/attest.conf)
+
+# For the time being, run `attest` as if it were a single test.
+set(disabled_tests)
+list(APPEND disabled_tests "mpirun=[5-9]")
+list(APPEND disabled_tests "explicit_ex1_2d.mpirun=4.input")
+list(APPEND disabled_tests "explicit_ex4_3d.mpirun=4.input")
+list(APPEND disabled_tests "explicit_ex8_2d.scratch_hier.input")
+list(APPEND disabled_tests "explicit_ex5_3d.mpirun=2.input")
+list(APPEND disabled_tests "explicit_ex2_3d.nodal_quadrature.input")
+list(APPEND disabled_tests "explicit_ex8_2d.input")
+list(APPEND disabled_tests "nwt_cylinder|rotating_barge|cib_double_shell")
+list(APPEND disabled_tests "free_falling_cyl_cib|cib_plate")
+list(JOIN disabled_tests "|" disabled_test_regex)
+add_test(
+  NAME autotests
+  COMMAND "${CMAKE_SOURCE_DIR}/attest"
+    "-j4"
+# Uncomment these once IBAMR/IBAMR#1293 is merged.
+#   "--verbose"
+#   "--test-timeout=60"
```

Comment:
I only made the test timeout that low because I wanted to debug the test execution - we should probably use a higher value in practice.

Reply:
OK, I've bumped it to 600 (10 minutes).

Do you have an idea of how long `attest` should take on the github runners (assuming excluded tests are not run)? I've overriden the default 10-minute timeout ctest uses to 30 minutes, but will change it if I need to.

Reply:
I'd estimate it should take 8 minutes. My 8 year old desktop with 4 cores (run with -j8 to mimic the increased load) takes 4 minutes (wall time); multiplying by 2 gives me my guess. Hence 10 is probably enough.

I don't know how performant the CI machines are so that might still be too little time.